fem/fespace.cpp

```diff
@@ -3936,6 +4253,57 @@ void FiniteElementSpace::Update(bool want_transform)
    }
 }
 
+void FiniteElementSpace::PRefineAndUpdate(const Array<pRefinement> & refs,
+                                          bool want_transfer)
+{
+   MFEM_VERIFY(PRefinementSupported(),
+               "p-refinement is not supported in this space");
+
+   if (want_transfer)
+   {
+      fesPrev.reset(new FiniteElementSpace(mesh, fec));
```

Comment:
I think this should account for `vdim` and `ordering`.

Reply:
Fixed in 7fe4c3c19943a1b5064aa33e99b357f41dbb75c7